### DISCUSSION TITLE
Help! Error Message: Failure when receiving data from the peer on handle

### DISCUSSION THREAD

**Discussion by joannawooastro** [abraunegg/onedrive#2325] [Q&A] [answered]
When syncing large-ish files (~130MB) I consistently get the following error message
```
Uploading new file [filename deleted] ... 
Uploading  35% |ooooooooooooooo                         |   ETA   00:00:12 ERROR: OneDrive returned an error with the following message:
  Error Message: Failure when receiving data from the peer on handle [handle deleted]
  Calling Function: perform()
```
The progress would sometimes reach a smaller percentage, sometimes a bigger one.  Sometimes the file successfully syncs, but most of the time files of this size fail to sync.

Any help would be appreciated.

System: Ubuntu 20.04.5 LTS
onedrive client version: 2.4.23, installed using the repository https://download.opensuse.org/repositories/home:/npreining:/debian-ubuntu-onedrive/xUbuntu_20.04/

**Comment by abraunegg** [comment, marked as answer]
@joannawooastro 
> `Error Message: Failure when receiving data from the peer on handle`

You have an issue with your network and your connection between you and the OneDrive Service | OneDrive API

This is a network issue. Please investigate issues with your network.

**Reply by joannawooastro** [reply]
Yup, that was it.  Thanks.
